draft-wussler-openpgp-pqc.md

```diff
@@ -622,7 +622,7 @@ ECDH component of the composite algorithms.
 
 {{tab-ecdh-cfrg-artifacts}}, {{tab-ecdh-nist-artifacts}}, and
 {{tab-ecdh-brainpool-artifacts}} describe the ECC-KEM parameters and artifact
-lengths. The artefacts in {{tab-ecdh-cfrg-artifacts}} follow the encodings
+lengths. The artifacts in {{tab-ecdh-cfrg-artifacts}} follow the encodings
```

Comment:
I am not sure whether we have commited to BE or AE. In BE, "artefact" would be appropriate: https://grammarist.com/spelling/artefact-artifact/

Reply:
OK, great, thanks for these insights, DKG. Once the topmost change regarding "cryptanalytic" has been reverted, I will hit "approve".